src/test/groovy/io/cify/runner/utils/TaskPoolManagerTest.groovy

```diff
@@ -57,18 +57,20 @@ class TaskPoolManagerTest extends GroovyTestCase {
     void testRunInParallel() {
         addTestTasks(10)
         taskPoolManager.runTasksInParallel(THREADCOUNT)
-
+        sleep(5000)
         assert TestTask.threadsNames.size() == THREADCOUNT
     }
 
     void testRunCount() {
         addTestTasks(10)
         taskPoolManager.runTasksInParallel(1)
+        sleep(1000)
```

Comment:
Otherwise sleeps are not OK, would be good to comment, why these sleeps are left here, also would be better to have more dynamic sleep